### PR TITLE
Disable IsAtLeastLibgdiplus6 for Apple Silicon

### PR DESCRIPTION
**Pull request by sdmaclea**
Related to #49111.

~~I didn't make this an issue...  Perhaps that is preferred...~~

~~But perhaps there should be a separate test that Drawing is supported with an issue.~~

### CODE REVIEW

**Comment by sdmaclea** [comment]
Once #49155 merges I can add something like the diff below if we want.

```diff
diff --git a/src/libraries/System.Drawing.Common/tests/GdiplusTests.cs b/src/libraries/System.Drawing.Common/tests/GdiplusTests.cs
index 6336e33d520..c790c63b34a 100644
--- a/src/libraries/System.Drawing.Common/tests/GdiplusTests.cs
+++ b/src/libraries/System.Drawing.Common/tests/GdiplusTests.cs
@@ -7,6 +7,13 @@ namespace System.Drawing.Tests
 {
     public class GdiplusTests
     {
+        [ConditionalFact(typeof(PlatformDetection), nameof(PlatformDetection.IsOSX))]
+        [ActiveIssue("https://github.com/dotnet/runtime/issues/49111", typeof(PlatformDetection), nameof(PlatformDetection.IsNotMacOSAppleSilicon))]
+        public void IsDrawingSupported()
+        {
+            Assert.True(Helpers.GetIsDrawingSupported());
+        }
+
         [ConditionalFact(typeof(PlatformDetection), nameof(PlatformDetection.IsOSX), nameof(PlatformDetection.IsDrawingSupported))]
         public void IsAtLeastLibgdiplus6()
         {
```

File: src/libraries/System.Drawing.Common/tests/GdiplusTests.cs [hunk 1]
```diff
@@ -7,7 +7,7 @@ namespace System.Drawing.Tests
 {
     public class GdiplusTests
     {
-        [ConditionalFact(typeof(PlatformDetection), nameof(PlatformDetection.IsOSX))]
+        [ConditionalFact(typeof(PlatformDetection), nameof(PlatformDetection.IsOSX), nameof(PlatformDetection.IsDrawingSupported))]
```
Comment (safern) [inline src/libraries/System.Drawing.Common/tests/GdiplusTests.cs:10]: Can we instead disable for ARM64? I don't want to be in the case where the non arm mac's are refreshed and the libgdiplus dependency is messed up (not installed or something) and this test helps us catch that libgdiplus is there.

Comment (sdmaclea) [reply src/libraries/System.Drawing.Common/tests/GdiplusTests.cs:10]: sure

Comment (sdmaclea) [reply src/libraries/System.Drawing.Common/tests/GdiplusTests.cs]: done

**Review by safern — APPROVED** [review]
Thanks!

**Comment by danmoseley** [comment]
@sdmaclea I believe the condition in this attribute is backwards. The value indicates when the active issue applies. Eg 
```
        [ActiveIssue("https://github.com/dotnet/runtime/issues/38817", typeof(PlatformDetection), nameof(PlatformDetection.IsMonoInterpreter))]
```
disables the test only for the mono interpreter.

Not useful here, but here's a more common form
```
   [ActiveIssue("https://github.com/dotnet/runtime/issues/38998", TestPlatforms.OSX)]
```
is an OSX specific issue.

I put this in a review comment which I see above but very unfortunately I apparently did not hit the right button to make it appear so you saw it.

File: src/libraries/System.Drawing.Common/tests/GdiplusTests.cs [hunk 2]
```diff
@@ -8,6 +8,7 @@ namespace System.Drawing.Tests
     public class GdiplusTests
     {
         [ConditionalFact(typeof(PlatformDetection), nameof(PlatformDetection.IsOSX))]
+        [ActiveIssue("https://github.com/dotnet/runtime/issues/49111", typeof(PlatformDetection), nameof(PlatformDetection.IsNotMacOSAppleSilicon))]
```
[the remark this answers is not in the document]
Comment (danmoseley) [reply src/libraries/System.Drawing.Common/tests/GdiplusTests.cs]: this was an older comment

**Comment by sdmaclea** [comment]
The condition felt backwards to me to. When the syntax was recommended https://github.com/dotnet/runtime/pull/49155#discussion_r587757748, it felt wrong, but neglected to ask...

Perhaps `ConditionalFact` uses the condition in the opposite sense than `ActiveIssue`.  I guess that makes sense.  For `ConditionalFact` the condition is when to execute a test.  For `ActiveIssue` the condition describes when to disable a test.

I need to revise this, #49165 & #49155.

Thanks for letting me know @danmoseley (with 2 `e`s 😉 )

**Comment by danmoseley** [comment]
I find these things hard to remember so I just grep for other ones...

**Comment by safern** [comment]
Good catch @danmoseley -- clearly I was also confused by this 😆

**Comment by sdmaclea** [comment]
The problem was this pattern was not used yet...  So when I grepped I found the `ConditionalFact` pattern which apparently we are deprecating at least for `ActiveIssue`s.

**Pull request merged**